### ISSUE TITLE
Highlight current day

### ISSUE THREAD

**Issue by varlesh** [varlesh/org.kde.plasma.digitalclock.wl#1]

**Comment by varlesh** [comment]
@psifidotos Hi Michail, sorry for disturb you again))  I'm fork original plasma widget Digital Clock and hide lines on calendar. But Will be good if you help me change color on calendar for use highlight color.
PS: my QML skills bad))

**Comment by varlesh** [comment]
For example on materia theme
Now drawed this:
![image](https://user-images.githubusercontent.com/8083855/60201348-20202680-9859-11e9-9fff-8a97c9d11b47.png)
But need replace color background to highlight on current day:
![123](https://user-images.githubusercontent.com/8083855/60201575-9e7cc880-9859-11e9-8ff5-b60c36df31c5.png)

**Comment by psifidotos** [comment]
where is the code ?

**Comment by psifidotos** [comment]
the mentioned code is in plasma-frameworks: https://github.com/KDE/plasma-framework/blob/master/src/declarativeimports/calendar/qml/DayDelegate.qml

it can not be forked easily because this is provided by calendar library...

the color of that rectangle is found at line #83 :::

for current date the background is set as:
theme.textColor with opacity:0.4 and when it is selected with opacity: 0.6

I suppose you prefer to be set to theme.buttonFocusColor OR theme.highlightColor BUT theme.highlightColor is already set of on mouse hovering states...

May I ask you one personal question?
Why not discussing this with plasma Visual Design Group? 
As the Papirus author VDG would be more than grateful to become part of it. In Plasma VDG some members are providing mockups for different parts of Plasma to be discussed and if Plasma VDG decides that a change must be done then plasma developers implement it...

For example: https://phabricator.kde.org/T10201 (VDG is discussing the window title bars in Breeze theme)

**Comment by psifidotos** [comment]
@varlesh ok sorry :) now I saw that you have forded an another digital clock :)

**Comment by psifidotos** [comment]
ok I confirmed if your forked version the comments above are valid, the original author was able to remove the border lines because MonthView has a property with name borderOpacity, but for defining colors for today, selected , hovered days there is NO option

**Comment by varlesh** [comment]
@psifidotos Yes, i'm fork this widget for this hacks only. Earlier i'm develop some icons on KDE (clementine, krusader, android-file-transfer, emacs and etc). But i'm not want destroy them HIG...
It's alternative widget without lines and hilight color

**Comment by varlesh** [comment]
> ok I confirmed if your forked version the comments above are valid, the original author was able to remove the border lines because MonthView has a property with name borderOpacity, but for defining colors for today, selected , hovered days there is NO option

It's impossible fixed and replace  `theme.textColor` to  `theme.highlightColor`?

**Comment by psifidotos** [comment]
> It's impossible fixed and replace `theme.textColor` to `theme.highlightColor`?

only inside plasma frameworks code

**Comment by varlesh** [comment]
it's pity. Thx for your info :)

**Comment by psifidotos** [comment]
@varlesh I think I can found a solution :) . I think that if we fork just 2 files for PlasmaCalendar library then we can do whatever we want with that layout...

I am thinking that no border and circular highlight might be better :)

Can you create mockups? Important thing is how to distinguish TODAY date from SELECTED and HOVERED dates...

I can make the implementation for my analog clock fork at https://github.com/psifidotos/applet-latte-analogclock and afterwards its pretty easy to merge the changes for your fork

**Comment by psifidotos** [comment]
just playing with this, we can do whatever you want :) create beautiful mockups for TODAY, SELECTED, HOVERED and I can reproduce them...

confirming that this is possible, I have reached this:

![image](https://user-images.githubusercontent.com/2216474/61066106-9f8f2780-a40d-11e9-9d73-1bce4b033f2c.png)

**Comment by varlesh** [comment]
Yes, it's looking very good. But i think better use light numbers for TODAY

**Comment by psifidotos** [comment]
Play with colors please and send mockups ... I used theme.highlightedTextColor and and theme.textColor but it didnt look as good from contrast point of view I think :

![image](https://user-images.githubusercontent.com/2216474/61066810-ec273280-a40e-11e9-8842-8272872975ea.png)

**Comment by varlesh** [comment]
Also see,  bottom widget not draw  maybe it,s possible fixed for dock?

**Comment by psifidotos** [comment]
> Also see, bottom widget not draw maybe it,s possible fixed for dock?

I can not understand, what bottom widget?

**Comment by varlesh** [comment]
See your screen - widget cut off on bottom. (beside with dock)

![image](https://user-images.githubusercontent.com/8083855/61067683-2f869e80-a419-11e9-93c6-51001503f300.png)

**Comment by psifidotos** [comment]
unfortunately Plasma does not have a way to solve this yet and I can not do anything

**Comment by varlesh** [comment]
My example:
 - Today 100% Highlight Text color with bold font on numbers
- Selected 50% Highlight Text color with semi-bold font on numbers
- Hover 20% Highlight Text color with semi-bold font on numbers

**Comment by varlesh** [comment]
> unfortunately Plasma does not have a way to solve this yet and I can not do anything

ok, i'm only ask about that))

**Comment by varlesh** [comment]
@psifidotos i'm found widget without cut off - Simple Menu:
https://github.com/KDE/plasma-simplemenu
![image](https://user-images.githubusercontent.com/8083855/61103747-529a6800-a484-11e9-9846-23a9821b9ef5.png)

**Comment by psifidotos** [comment]
@varlesh it's developer is using its own Plasma Dialog in order to achieve this
